src/modes/replay.ts

```diff
@@ -0,0 +1,9 @@
+import { ReplayBlockOptions, ReplayTxOptions } from "../types";
+
+export async function replayBlock(_options: ReplayBlockOptions) {
```

Comment:
nit: just a suggestion, but since there's various commands, I would make a dir commands and one file per command. 
Here replay defines 2 commands, and there might be commands that are not in the "modes" dir, so it feels easier to mantain if theres 1:1 mapping between command and file

Reply:
Yep, I think this is where things will naturally converge. Great idea. Tracking in #60.